src/sm.c

```diff
@@ -221,13 +222,13 @@ static SM_RESULT sm_close_begin_internal(SM_HANDLE sm)
                     {
                         /*Codes_SRS_SM_02_071: [ If there are any failures then sm_close_begin shall fail and return SM_ERROR. ]*/
                         LogError("sm name=%s. failure in InterlockedHL_WaitForValue(&sm->non_barrier_call_count=%p, 0, INFINITE), state was %" PRI_SM_STATE "", sm->name, &sm->non_barrier_call_count, SM_STATE_VALUE(state));
```

Comment:
>INFINITE [](start = 119, length = 8)

UINT32_MAX

Reply:
Fixed.

---
In reply to: [501166222](https://github.com/Azure/azure-c-util/pull/66#discussion_r501166222) [](ancestors = 501166222)